pkg/test/context.go

```diff
@@ -24,11 +24,16 @@ import (
 	"k8s.io/client-go/restmapper"
 )
 
+// todo(camilamacedo86): Rename TestCtx to Context
+// todo(camilamacedo86): need to be done before: 1.0.0
+// todo(camilamacedo86): - remove all deprecated notes
+// todo(camilamacedo86): - remove TestCtx struct
+// todo(camilamacedo86): - remove newTestCtx func
+// todo(camilamacedo86): - remove //nolint:golint and //nolint:staticcheck annotations
+
+// Deprecated: TestCtx exists for historical compatibility
+// Yoou should use Context instead of.
 type TestCtx struct { //nolint:golint
```

Comment:
Switch these so that the struct type is `Context` and the alias is `TestCtx`. Hopefully that removes the need for all the nolint comments, except for on the alias itself.

```suggestion
type Context struct {
```